### ISSUE TITLE
feature: remove connect button, add status indicator

### ISSUE THREAD

**Issue by stefanix** [stefanix/LasaurApp#38]
There is no good reason to have a connect button. In most cases one wants to be connected anyways and this is what LasaurApp should try implicitly. For the rare cases when one wants to be disconnected closing LasaurApp will be more appropriate.

There should still be some indicator to show connection status. This can probably be combined with other status indicators (chiller off, door open, limit hit, ready)

**Comment by javl** [comment]
I disagree; there is no reason to remove this button and I'm sure it will only introduce a lot of frustration when you just see the status and there is no way to change it.
